systemvm/agent/noVNC/app/ui.js

```diff
@@ -471,22 +474,31 @@ const UI = {
         clearTimeout(UI.statusTimeout);
 
         switch (statusType) {
+            case 'encrypted':
```

Comment:
we need to re-apply these changes when upgrade novnc.
not a big issue, just need to pay a bit more attention